### PR TITLE
9427 DEV | Add notification indicator for pending approvals

### PR DESCRIPTION
**Pull request by skilledDev96**
Added MatBadge to Employees Tab on NavBar for Pending Approvals

[Azure Ticket](https://dev.azure.com/retro-rabbit/RetroGradOnboard/_boards/board/t/RetroGradOnboard%20Team/Stories?System.AssignedTo=rhart%40retrorabbit.co.za&workitem=9427)

image

### CODE REVIEW

File: src/app/components/hris/employees/new-employee/new-employee.component.ts [hunk 1]
```diff
@@ -133,7 +133,7 @@ export class NewEmployeeComponent implements OnInit {
       .subscribe((data: EmployeeProfile[]) => {
         this.Employees = data;
       });
-    this.navService.hideNav();
+      // this.navService.hideNav();
```
Comment (Juanro-Minne) [inline src/app/components/hris/employees/new-employee/new-employee.component.ts:136]: you might have missed this 👍🏻

File: src/app/services/hris/employee/employee-profile.service.ts [hunk 2]
```diff
@@ -25,6 +25,7 @@ export class EmployeeProfileService {
               private employeeStore:Store<{employees:EmployeeState}>, 
               private cookieService: CookieService) { }
 
+  //Endpoint not working, ID expected.
```
Comment (tbddorothy) [inline src/app/services/hris/employee/employee-profile.service.ts:28]: What's happening here?

**Comment by tbddorothy** [comment]
Please add screenshots of the work

File: src/app/services/hris/employee/employee-profile.service.ts (hunk 2 above)
Comment (skilledDev96) [reply src/app/services/hris/employee/employee-profile.service.ts:28]: Call seems Redundant?
Endpoints that get the EmployeeProfile is Expecting an ID, email or we just retrieve all.

**Review by Juanro-Minne — DISMISSED** [review]

**Review by DewanAlberts — APPROVED** [review]

**Review by ReinoRabbit — DISMISSED** [review]

**Review by Juanro-Minne — DISMISSED** [review]

File: src/app/components/hris/admin-dashboard/admin-dashboard.component.ts [hunk 3]
```diff
@@ -18,7 +18,7 @@ import { Router } from '@angular/router';
 import { AuthAccessService } from 'src/app/services/shared-services/auth-access/auth-access.service';
 import { EmployeeCountDataCard } from 'src/app/models/hris/employee-count-data-card.interface';
 import { ChurnRateDataCard } from 'src/app/models/hris/churn-rate-data-card.interface';
-import { ChartComponent } from '../charts/charts.component';
+import { EmpBankStarterService } from 'src/app/services/hris/employee/employee-bankingandstarterkit.service';
```
Comment (tbddorothy) [inline src/app/components/hris/admin-dashboard/admin-dashboard.component.ts:21]: Please refrain from abbreviating names

File: src/app/components/hris/admin-dashboard/admin-dashboard.component.ts [hunk 4]
```diff
@@ -90,6 +90,7 @@ export class AdminDashboardComponent {
   rolesSelected: string[] = [];
   categoriesSelected: string[] = [];
   constructor(
+    private empBankStarterService: EmpBankStarterService,
```
Comment (tbddorothy) [inline src/app/components/hris/admin-dashboard/admin-dashboard.component.ts:93]: Ditto

File: src/app/components/hris/employees/employee-approvals/employee-approvals.component.ts [hunk 5]
```diff
@@ -8,23 +8,22 @@ import { MatSort, Sort } from '@angular/material/sort';
 import { dataTypes } from 'src/app/models/hris/constants/types.constants';
 import { Dialog } from 'src/app/models/hris/confirm-modal.interface';
 import { EmployeeDocument } from 'src/app/models/hris/employeeDocument.interface';
-import { EmployeeBankingandstarterkitService } from 'src/app/services/hris/employee/employee-bankingandstarterkit.service';
-import { EmployeeBanking } from 'src/app/models/hris/employee-banking.interface';
+import { EmpBankStarterService } from 'src/app/services/hris/employee/employee-bankingandstarterkit.service';
```
Comment (tbddorothy) [inline src/app/components/hris/employees/employee-approvals/employee-approvals.component.ts:11]: Ditto

**Review by tbddorothy — APPROVED** [review]

**Review by ReinoRabbit — APPROVED** [review]

**Review by Juanro-Minne — APPROVED** [review]

**Pull request merged**